src/main/resources/com/microsoft/azure/vmagent/AzureVMAgentTemplate/config.jelly

```diff
@@ -36,9 +36,18 @@
                 <f:select/>
             </f:entry>
 
-            <f:entry title="${%Storage_Account_Name}" field="storageAccountName"
-                     help="/plugin/azure-vm-agents/help-storageAccountName.html">
-                <f:textbox/>
+            <f:entry title="${%Storage_Account_Name}">
+                <f:radioBlock name="storageAccountNameReferenceType" value="new" title="${%Storage_Account_Create_New}" checked="${instance == null || instance.isCreateNewStorageAccount('new')}" inline="true">
```

Comment:
I reference the usage in sample and previous codes, but I found `Boolean` is also work.